IPs/RSKIP434.md

```diff
@@ -0,0 +1,86 @@
+---
+rskip: 434
+title: Bridge Bitcoin block chainwork up to 12 Bytes
+created: 26-Jun-24
+author: JD,JZ,MI
+purpose: Sec
+layer: Core
+complexity: 1
+status: Draft
+description: Change in Bridge Bitcoin storage to accept blocks with chainwork serialized up to 12 Bytes
```

Comment:
just a suggestion: i would say `12 _unsigned_ bytes` to be more specific

Reply:
Agree